server/api.go

```diff
@@ -80,6 +80,25 @@ func (d dexAPI) DeleteClient(ctx context.Context, req *api.DeleteClientReq) (*ap
 	return &api.DeleteClientResp{}, nil
 }
 
+func (d dexAPI) GetClient(ctx context.Context, req *api.GetClientReq) (*api.GetClientResp, error) {
+	c, err := d.s.GetClient(req.Id)
+	if err != nil {
+		d.logger.Errorf("api: failed to get client: %v", err)
+		return nil, fmt.Errorf("get client: %v", err)
+	}
+	return &api.GetClientResp{
+		Client: &api.Client{
+			Id:           c.ID,
+			Secret:       c.Secret,
```

Comment:
Who calls this?  Is it safe to return all the information including secret?

Reply:
All grpc API are not used now. It's prepared for cluster-admin at first.